src/GPT2-Simple.ipynb

```diff
@@ -1 +1,991 @@
+      "\n",
+      "[1510 | 1408.55] loss=2.26 avg=2.62\n",
+      "[1520 | 1417.25] loss=1.90 avg=2.61\n",
+      "[1530 | 1426.08] loss=2.85 avg=2.61\n",
+      "[1540 | 1434.78] loss=2.36 avg=2.61\n",
+      "[1550 | 1443.43] loss=2.67 avg=2.61\n",
+      "[1560 | 1452.10] loss=2.68 avg=2.61\n",
+      "[1570 | 1460.79] loss=2.79 avg=2.61\n",
+      "[1580 | 1469.46] loss=2.76 avg=2.61\n",
+      "[1590 | 1478.17] loss=2.32 avg=2.61\n",
+      "[1600 | 1486.86] loss=1.86 avg=2.60\n",
+      "[1610 | 1495.53] loss=2.48 avg=2.60\n",
+      "[1620 | 1504.21] loss=2.22 avg=2.59\n",
+      "[1630 | 1512.87] loss=2.53 avg=2.59\n",
+      "[1640 | 1521.57] loss=2.77 avg=2.60\n",
+      "[1650 | 1530.27] loss=2.15 avg=2.59\n",
+      "[1660 | 1538.93] loss=2.67 avg=2.59\n",
+      "[1670 | 1547.58] loss=2.22 avg=2.59\n",
+      "[1680 | 1556.24] loss=1.68 avg=2.58\n",
+      "[1690 | 1564.92] loss=1.78 avg=2.57\n",
+      "[1700 | 1573.60] loss=2.07 avg=2.56\n",
+      "[1710 | 1582.27] loss=1.67 avg=2.55\n",
+      "[1720 | 1590.97] loss=1.80 avg=2.54\n",
+      "[1730 | 1599.66] loss=2.04 avg=2.53\n",
+      "[1740 | 1608.34] loss=2.13 avg=2.53\n",
+      "[1750 | 1617.01] loss=2.27 avg=2.53\n",
+      "[1760 | 1625.66] loss=1.95 avg=2.52\n",
+      "[1770 | 1634.32] loss=2.40 avg=2.52\n",
+      "[1780 | 1642.99] loss=2.40 avg=2.52\n",
+      "[1790 | 1651.66] loss=2.50 avg=2.52\n",
+      "[1800 | 1660.33] loss=2.37 avg=2.51\n",
+      "[1810 | 1668.99] loss=2.58 avg=2.52\n",
+      "[1820 | 1677.66] loss=2.47 avg=2.51\n",
+      "[1830 | 1686.33] loss=2.05 avg=2.51\n",
+      "[1840 | 1695.03] loss=2.77 avg=2.51\n",
+      "[1850 | 1703.73] loss=1.72 avg=2.50\n",
+      "[1860 | 1712.40] loss=2.54 avg=2.50\n",
+      "[1870 | 1721.07] loss=2.69 avg=2.51\n",
+      "[1880 | 1729.93] loss=2.03 avg=2.50\n",
+      "[1890 | 1738.61] loss=1.62 avg=2.49\n",
+      "[1900 | 1747.25] loss=2.11 avg=2.48\n",
+      "[1910 | 1755.94] loss=2.06 avg=2.48\n",
+      "[1920 | 1764.62] loss=2.32 avg=2.48\n",
+      "[1930 | 1773.30] loss=2.20 avg=2.47\n",
+      "[1940 | 1781.98] loss=1.68 avg=2.47\n",
+      "[1950 | 1790.68] loss=2.54 avg=2.47\n",
+      "[1960 | 1799.33] loss=2.63 avg=2.47\n",
+      "[1970 | 1808.00] loss=2.52 avg=2.47\n",
+      "[1980 | 1816.68] loss=2.01 avg=2.46\n",
+      "[1990 | 1825.36] loss=2.10 avg=2.46\n",
+      "[2000 | 1834.06] loss=2.11 avg=2.46\n",
+      "Saving checkpoint/run1/model-2000\n",
+      "CPU times: user 21min 32s, sys: 6min 46s, total: 28min 19s\n",
+      "Wall time: 31min 47s\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "gpt2.finetune(sess,\n",
+    "              dataset=file_name,\n",
+    "              model_name='355M',\n",
+    "              steps=2000,\n",
+    "              restore_from='fresh', #change to 'latest' if restarting fine-tuning from saved checkpoint\n",
+    "              checkpoint_dir = \"drive/My Drive/gpt2/checkpoint\", #if wanting to save checkpoints on Drive\n",
```

Comment:
Fourth change: added parameter `checkpoint_dir` with an example path to one's Drive